packages/sail_ui/lib/widgets/containers/single_value_container.dart

```diff
@@ -33,10 +33,11 @@ class SingleValueContainer extends StatelessWidget {
           const SizedBox(
             width: 13,
           ),
-        SizedBox(
-          width: width,
-          child: SailText.secondary12(label),
-        ),
+        if (label != null)
```

Comment:
Do we really want this? Should figure out a fitting label instead imo. Can't see it used anywhere

Reply:
I use it for the yet-to-be-bundled withdrawals